inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AgentServiceImpl.java

```diff
@@ -518,6 +519,11 @@ private DataConfig getDataConfig(StreamSourceEntity entity, int op) {
                     topicConfig.setInlongGroupId(groupId);
                     topicConfig.setTopic(mqResource);
                     dataConfig.setTopicInfo(topicConfig);
+                } else if (MQType.KAFKA.equals(mqType)) {
+                    DataProxyTopicInfo topicConfig = new DataProxyTopicInfo();
+                    topicConfig.setInlongGroupId(groupId);
+                    topicConfig.setTopic(groupId + DOT + streamId);
```

Comment:
@haifxu Can you please check how Kafka topics are spliced in InLong?

Reply:
`InlongGroupInfo.mqResource` + dot + `InlongStreamInfo.mqResource`.
So the default value should be `groupId` + dot + `streamId`.

Reply:
It may be better to use `groupEntity.getMqResource()` + DOT + `streamEntity.getMqResource()` in this place.

Reply:
> It may be better to use `groupEntity.getMqResource()` + DOT + `streamEntity.getMqResource()` in this place.

cc @wangpeix

Reply:
Okay, thanks. I'll optimize it.